src/utils/helpers/priorities.js

```diff
@@ -2,11 +2,18 @@ export const IMMEDIATE_PRIORITY_CODE = 1
 export const EMERGENCY_PRIORITY_CODE = 2
 export const URGENT_PRIORITY_CODE = 3
 export const NORMAL_PRIORITY_CODE = 4
+export const PLANNED_PRIORITY_CODE = 9
+export const PRIORITY_CODES_WITHOUT_DRS = [PLANNED_PRIORITY_CODE]
 
 export const HIGH_PRIORITY_CODES = [
   IMMEDIATE_PRIORITY_CODE,
   EMERGENCY_PRIORITY_CODE,
 ]
+export const LOW_PRIORITY_CODES = [
+  URGENT_PRIORITY_CODE,
+  NORMAL_PRIORITY_CODE,
+  PLANNED_PRIORITY_CODE,
+]
```

Comment:
Please delete this - you weren't to know but since this was implemented in this branch, develop has been updated so that this const is not needed anywhere any more. (Basically, we list only the high priority codes now and assume everything else is low priority, which means new codes added to the backend shouldn't need frontend updates 🤞 .)